spec/support/shared_examples_policy_machine.rb

```diff
@@ -621,12 +640,16 @@
         [@u2, @w, @draft_u2], [@u2, @r, @trash_u2], [@u2, @w, @trash_u2]
       ]
 
-      # TODO:  remove the expected privilege below once prohibitions are put in place.
-      # In the example, @u2 is prohibited from writing to @in_u2
-      expected_privileges << [@u2, @w, @in_u2]
-
       assert_pm_privilege_expectations(policy_machine.privileges, expected_privileges)
     end
+
+    it 'can ignore prohibitions' do
+      expect(policy_machine.is_privilege_ignoring_prohibitions?(@u2, @w, @in_u2)).to be
+      ignoring_prohibitions = policy_machine.scoped_privileges(@u2, @in_u2, ignore_prohibitions: true).map{ |_,op,_| op.unique_identifier }
+      with_prohibitions = policy_machine.scoped_privileges(@u2, @in_u2).map{ |_,op,_| op.unique_identifier }
+      expect(ignoring_prohibitions - with_prohibitions).to eq([@w.unique_identifier])
+    end
+
   end
 
```

Comment:
`is_privilege` doesn't take an ignore_prohibitions option, that's just a separate method that we do have specs for.